### PR TITLE
Add recipe for octave_kernel

### PR DESCRIPTION
**Pull request by blink1073**
Installs the kernel in post-link stage.

### CODE REVIEW

File: recipes/octave_kernel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  license_family: BSD
+
+
+extra:
+    recipe-maintainers:
+        - blink1073
+
+# See
+# http://docs.continuum.io/conda/build.html for
+# more information about meta.yaml
```
Comment (scopatz) [inline recipes/octave_kernel/meta.yaml]: Can you please remove these comments? Thanks!

Comment (blink1073) [inline recipes/octave_kernel/meta.yaml]: Stricken from the record.

**Comment by scopatz** [comment]
Also @blink1073, can you please enable CircleCI for your fork of staged recipes? Thanks! The PR won't run on Linux without it.

**Comment by blink1073** [comment]
Done.

**Pull request merged**